auxx/src/Command/Types.hs

```diff
@@ -52,7 +51,7 @@ data Command
     | Send Int (NonEmpty TxOut)
     | SendToAllGenesis !SendToAllGenesisParams
     | Vote Int Bool UpId
-    | ProposeUpdate !ProposeUpdateParams
+    | ProposeUpdate !Int !ProposeUpdateParams !Bool
```

Comment:
Why are these values (`Int` and `Bool`) not in `ProposeUpdateParams`?

Reply:
I assumed `ProposeUpdateParams` contains related to `UpdateProposal` fields, like software version and etc.

Reply:
Not really, `ProposeUpdateParams` contains all parameter for `ProposeUpdate` command. That's what the documentation says.

Reply:
Also it's better to do as little changes as possible in release branches, because otherwise merging will be harder.

Reply:
>merging will be harder.
pffff, who cares :yao: